doc/source/conf.py

```diff
@@ -473,3 +474,11 @@ class NumPyLexer(CLexer):
             inherit,
         ],
     }
+
+
+# -----------------------------------------------------------------------------
+# Breathe & Doxygen
+# -----------------------------------------------------------------------------
+breathe_projects = dict(numpy=os.path.join("..", "build", "doxygen", "xml"))
+breathe_default_project = "numpy"
+breathe_default_members = ("members", "undoc-members", "protected-members")
```

Comment:
Do we need to generate documentation for undoc-members?

Reply:
operators, overloaded and obvious methods may not contain any comment block, and under any circumstances, any public, and protected class/struct members should be exposed in the document and if there is a necessity to hide any members then tag `@cond IGNORED` `@endcond`  can be used.